codan/org.eclipse.cdt.codan.ui.cxx/plugin.xml

```diff
@@ -8,6 +8,14 @@
             class="org.eclipse.cdt.codan.internal.ui.cxx.Startup"></startup>
    </extension>
 
+   <!-- Suppress startup in org.eclipse.ease.runScript application (bug 580960) -->
+   <extension
```

Comment:
[Sorry, I missed the direction on this discussion during the last CDT call]

I still think EASE need to stop invoking IStartup code. We should not encourage misuse of the Platform.

Reply:
💯 what EASE is doing is bad idea. They should ditch the whole IStartup and, in order of my preference do one of these:

- Take the IStartup they want access to and use a different non-UI startup mechanism, this probably requires contributions to whatever upstream project they have dependency on.
- Create a new EASE Startup extension point that will be run that plug-ins can opt-in to
- Provide a way in the script to explicitly run the IStartup of a bundle they care about, e.g in the script.

```python
runEclipseUIStart("name.of.bundle")
```
- Make it a "whitelist" so that plug-ins opt-in to the behaviour instead of opting out.

But, until this is resolved we need to have a non-broken CDT + EASE integration.